js/main.js

```diff
@@ -24,15 +30,37 @@ function onPlayerReady (event) {
 // 5. The API calls this function when the player's state changes.
 var done = false;
 function onPlayerStateChange (event) {
-  if (event.data == YT.PlayerState.ENDED) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["YT" is not defined.](https://www.codacy.com/app/j-hasseriis/cepheus/file/1857423618/issues/source?bid=3085146&fileBranchId=3085654#l33)